docs/configuring-os2loop.md

```diff
@@ -24,10 +24,20 @@ In order to make the media library work, go to
 media categories for the site.
 
 Go to `/admin/structure/taxonomy` and make sure that the Profession and Subject
-taxonomies containt the terms needed for the content. Users can dynamically
+taxonomies contain the terms needed for the content. Users can dynamically
 create new terms in the Tag taxonomy, but any default terms can be added if
 needed.
 
+All taxonomy terms are added and edited on '/admin/structure/taxonomy'
```

Comment:
For consistency with other admin paths in the documentation you should enclose the path in backticks, i.e. `` `/admin/structure/taxonomy` `` (cf. https://en.wikipedia.org/wiki/Backtick).